regparser/federalregister.py

```diff
@@ -25,7 +31,10 @@ def fetch_notice_json(cfr_title, cfr_part, only_final=False,
         params["conditions[type][]"] = 'RULE'
     if max_effective_date:
         params["conditions[effective_date][lte]"] = max_effective_date
-    response = requests.get(API_BASE + "articles", params=params).json()
+    url = API_BASE + "articles"
+    logger.info("Fetching notices - URL: %s Params: %r", url, params)
```

Comment:
This is certainly helpful to log, but two things to keep in mind:

1. requests includes the domain name when logging is set to `debug`
1. the request might be coming from a local cache (`requests_cache`). If we're adding logging, we should probably indicate if the request is actually being executed